src/scripts/5-approve-modules.ts

```diff
@@ -0,0 +1,19 @@
+import { ContractRunner } from 'ethers-v6';
+import { address } from '../constants';
+import { getAccountingExtension, getSigner } from '../helpers';
+import { getNetworkName } from '../utils';
+
+/**
+ * Approve modules script
```

Comment:
This script should be the very first one, because the rest won't work until users set their approvals.
```suggestion
 * Approve Modules script
```